src/modules/Lan.rb

```diff
@@ -667,18 +601,10 @@ def Import(settings)
     # @return dumped settings
     def Export
       profile = Y2Network::AutoinstProfile::NetworkingSection.new_from_network(yast_config)
-      ay = {
-        "dns"                  => profile.dns&.to_hashes || {},
-        "net-udev"             => profile.udev_rules&.udev_rules&.map(&:to_hashes) || [],
-        "s390-devices"         => profile.s390_devices&.to_hashes&.fetch("devices", []) || [],
-        "config"               => NetworkConfig.Export,
-        "interfaces"           => profile.interfaces&.interfaces&.map(&:to_hashes) || [],
-        "ipv6"                 => @ipv6,
-        "routing"              => profile.routing&.to_hashes || {},
-        "managed"              => NetworkService.is_network_manager,
-        "start_immediately"    => autoinst.start_immediately,
-        "keep_install_network" => autoinst.keep_install_network
-      }
+      ay = profile.to_hashes
+      ay["ipv6"] = @ipv6
```

Comment:
So at some point, we should move the IPv6 settings into the Config class, right?

Reply:
Completely agree, I kept it as it is by now, but should be moved ASAP.